topics/imaging/tutorials/detection-of-mitoflashes/tutorial.md

```diff
@@ -0,0 +1,152 @@
+>    - **Starting time point** (`frame_1st`): `1` (first frame in the stack)
+>    - **Ending time point** (`frame_end`): `0` (use `0` to process until the last frame)
+>    - **Intensity measurement method** (`typ_intens`): Set to `Smoothed` for a more averaged intensity or `Robust` for resilience to outliers.
+>    - **Threshold** (`thres`): A starting value of `10%` of the global maximum intensity is often effective, but increasing to `15-20%` is possible for higher sensitivity.
+>    - **Gaussian filter sigma** (`ssig`): Use a value around `1` to suppress noise; increase to `1.5` if noise is high.
+>    - **Boundary pixels** (`bndy`): Set to `10` to ignore spots within the image edges, adjusting as needed for image size.
+>    This tool detects mitochondria as spots or 'mitoflashes' based on intensity maxima, identifying them as distinct regions. Adjusting the threshold parameter can improve the accuracy of spot detection, allowing for better identification of mitoflash events.
+>
+{: .hands_on}
+
+## Step 2: Tracking Mitochondrial Movement
+
+> <hands-on-title>Tracking Mitochondrial Movement</hands-on-title>
+>
+> 1. {% tool [Perform linking in time series (nearest neighbors)](toolshed.g2.bx.psu.edu/repos/imgteam/points_association_nn/ip_points_association_nn/0.0.3-2) %} with the following recommended parameters:
+>    - {% icon param-file %} *"Coordinates (and intensities) of input points"*: Output from the **Spot Detection** tool.
+>    - **Neighborhood size** (`nbpx`): Set to `6` pixels as a starting value. Adjust within `1-10` pixels based on the density of points and proximity between spots across frames.
+>    - **Intensity threshold** (`thres`): A recommended starting value is `25%` of the global maximum intensity. Increase to `30-40%` for stricter filtering if spots have low intensity.
+>    - **Minimum track length** (`minlen`): Use `50%` of the sequence length as a baseline. Adjust based on the expected duration of mitoflash events.
+>    
+>    This tool links detected spots across frames to track movement. The **Neighborhood size** parameter controls the pixel range for associating points between frames. Increasing the **Intensity threshold** filters out weaker signals, improving track accuracy.
+>
+{: .hands_on}
+
+# Detecting Mitoflashes
+
+Mitoflashes are identified based on sudden changes in fluorescence intensity in mitochondria, signifying superoxide bursts. We will use a curve-fitting tool to capture these fluctuations and measure their characteristics.
+
+## Step 3: Curve Fitting for Mitoflash Detection
+
+> <hands-on-title>Mitoflash Detection</hands-on-title>
+>
+> 1. {% tool [Perform curve fitting](toolshed.g2.bx.psu.edu/repos/imgteam/curve_fitting/ip_curve_fitting/0.0.3-2) %} with the following parameters:
+>    - {% icon param-file %} *"File name of input data points (xlsx)"*: Select `spots_linked.xlsx`, the output file from the **Perform linking in time series (nearest neighbors)** tool.
```

Comment:
I think the file produced by the preceding tool does not have the name `spots_linked.xlsx` in general, so either you should describe that the output should be renamed, or simply write it analogously to how you did for the previous step:
```suggestion
>    - {% icon param-file %} *"File name of input data points (xlsx)"*: Output from the **Perform linking in time series (nearest neighbors)** tool.
```